### PR TITLE
Add additional documentation of what is expected when picking up requests

### PR DESCRIPTION
**Pull request by AdmiringWorm**
This additional documentation tries to details what is expected
of the maintainer when he/she is picking up a new request from the
repository. A notice about failure to follow the expected path
have also been added to the documentation.

/CC @pauby

### CODE REVIEW

File: README.md
Comment (pauby) [inline README.md:38]: Chocolatey instead of chocolatey?

File: README.md
Comment (pauby) [inline README.md:35]: Another blank line needed her?

File: README.md
Comment (pauby) [inline README.md:39]: Capital C in Chocolatey?

File: README.md
Comment (pauby) [inline README.md:43]: I think being specific with the time is better. We give maintainers at least 7 days to respond to a contact request on the site so I think asking 'at most' 7 days on this repo is consistent?

File: README.md
Comment (pauby) [inline README.md:45]: Typo 'ot' should be to.

And also "Use the 'Contact Site Admins' link on the package page, and point to ..."

**Review by pauby — CHANGES_REQUESTED** [review]
Just a few changes. Awesome work picking this up!

File: README.md
Comment (AdmiringWorm) [reply README.md:38]: it was actually meant to be `chocolatey.org` (forgot that last part).
Not sure what would be best, `Chocolatey` or `chocolatey.org`

File: README.md
Comment (AdmiringWorm) [reply README.md:35]: 1 blank line should be enough between a paragraph and a list.
Or do you mean separating this line into its own paragraph?

File: README.md
Comment (pauby) [reply README.md:38]: I think chocolatey.org

File: README.md
Comment (pauby) [reply README.md:35]: Yeah. It looked like you were putting that into a separate paragraph. If not, resolve this comment.

File: README.md
Comment (AdmiringWorm) [reply README.md]: Changed this to `chocolatey.org`

File: README.md
Comment (AdmiringWorm) [reply README.md]: Fixed :)

File: README.md
Comment (AdmiringWorm) [reply README.md]: I rewrote the last two lines a bit, explicitly mentioning 7 days as the limit.

File: README.md
Comment (AdmiringWorm) [reply README.md]: Although not part of the initial changes, I fixed this as well :smile:

**Comment by AdmiringWorm** [comment]
I added an additional note regarding removal of trusted package status for RFM requests.
I think it would be a good procedure to remove that as soon as possible.

**Comment by AdmiringWorm** [comment]
@pauby do you have the possibility to do an updated review of this PR?

File: README.md
Comment (pauby) [inline README.md:47]: Can we change the end of this line to be to" 'Contact Site Admins' link on the package page," - that just makes it clearer what link they need to click?

**Comment by pauby** [comment]
Looks good @AdmiringWorm - just one comment 😢

File: README.md
Comment (AdmiringWorm) [reply README.md:47]: Changing the `Contact Site Admins` to be in bold or in italic as well then perhaps.
So it is easier differentiated from the rest of the text?

ie
```suggestion
**Note**: If you are taking over maintenance for a package, you also need to be added to the list of maintainers. Use the **Contact Site Admins** link on the package page, point to the particular issue in this repository, and ask to be added.
```

or

```suggestion
**Note**: If you are taking over maintenance for a package, you also need to be added to the list of maintainers. Use the _'Contact Site Admins'_ link on the package page, point to the particular issue in this repository, and ask to be added.
```

Comment (AdmiringWorm) [reply README.md:47]: ^^ @pauby

Comment (pauby) [reply README.md:47]: I'd prefer bold?

Comment (AdmiringWorm) [reply README.md]: @pauby alright. I updated to `Contact Site Admins` and made it bold.

**Review by pauby — APPROVED** [review]

**Comment by pauby** [comment]
Not sure if you're going to add any more to this @AdmiringWorm - you happy for me to merge?

**Comment by AdmiringWorm** [comment]
nope, I did not plan to anything more. Feel free to merge it.

Once it is merged I will go through the existing RFM requests later today to ensure neither of them are set to trusted (as mentioned in the changes I made).

**Pull request merged**
